src/js/components/common/Header/UserInfo.tsx

```diff
@@ -1,17 +1,43 @@
-import { KebabMenu, Icon } from '..';
-
 import styles from './UserInfo.module.css';
-import React, { useContext } from 'react';
+import React, { useContext, useState, useRef } from 'react';
 import { userContext } from '../../../contexts/CurrentUser';
+import { useHistory } from 'react-router';
+import { logout } from '../../../API/auth/methods';
+import { FiLogOut } from 'react-icons/fi';
+import { Icon, KebabMenu } from '..';
+import { ControlledMenu, MenuItem } from '@szhsin/react-menu';
+import '@szhsin/react-menu/dist/index.css';
 
 const UserInfo = () => {
   const { user } = useContext(userContext);
+  const [isOpen, setOpen] = useState<boolean>(false);
+  const ref = useRef<null>(null);
```

Comment:
To można inaczej otypować, spróbuj `useRef<HTMLDivElement>(null)`. Jeśli nie zadziała to zostaw.

Reply:
Działa, zmieniłam